src/app/types/request-info-form.endpoint.ts

```diff
@@ -20,7 +20,7 @@ export interface RequestInfoForm {
   nonce: string
   clientHavePersistentTokens: boolean
   scopesAsString: string
-  error: 'login_required' | 'interaction_required'
```

Comment:
are the more errors here than just 'login_required' | 'interaction_required', can't we just list those as well?

Reply:
Yes, there are more. Ok, I will list them all